src/db/http/docusign-dao.js

```diff
@@ -25,7 +25,11 @@ const getEnvelopeDocumentTabs = async (params, query) => {
     if (pageNumbers) {
       uri = paramsLink(uri, { page_numbers: pageNumbers.join(',') });
     }
-    return await rp.get({ ...{ uri }, ...httpOptions });
+    const result = await rp.get({ ...{ uri }, ...httpOptions });
+    if (result instanceof Error) {
+      throw result;
```

Comment:
Their API doesn't throw an error, instead, it "return" an error object so the promise will be resolved no matter what (for API errors, but the URL error  e.g. invalid URL or invalid credentials will still be thrwon). I think it's better to throw an error as we usually do, so I throw it here myself lol.